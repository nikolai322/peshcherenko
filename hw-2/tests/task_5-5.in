5
-1 -2 -3 -4 -5
3
0.01 1.00 -0.01
