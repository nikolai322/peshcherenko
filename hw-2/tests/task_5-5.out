-15
1.000000
